public/usage-examples/physics/apply_matrix_to_quad/apply_matrix_to_quad-simple-oop.cs

```diff
@@ -0,0 +1,52 @@
+using static SplashKitSDK.SplashKit;
```

Comment:
For OOP usage examples are done with just `using SplashKitSDK;`
Meaning each splashkit function would use `SplashKit.`